### ISSUE TITLE
Configure default notification icon

### ISSUE THREAD

**Issue by foxanna** [Microsoft/AppCenter-SDK-Android#618]
**Problem:**

`PushNotifier.java` uses application launcher icon as notification icon in case no icon was sent inside the push.

According to Android guidelines for [Status Bar Icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_status_bar.html)

> "Status bar icons are composed simply of white pixels on a transparent backdrop..."

In case the app launcher icon is 100% opaque and is used as a notification icon, it is displayed as a plain white circle. This issue can be reproduced on devices with stock Android UI like Nexus. Xiaomi or Samsung phones display launcher icon in the status bar as expected.

**Possible solution:**

According to [Topic Messaging on Android](https://firebase.google.com/docs/cloud-messaging/android/topic-messaging) a native Android developer can add these lines inside the `Manifest` `application` tag to provide a custom default icon for push notification.
```
<meta-data
        android:name="com.google.firebase.messaging.default_notification_icon"
        android:resource="@drawable/ic_stat_ic_notification" />
```

As long as `AppCenter.Push` implementation uses Firebase, it would be very helpful if `PushNotifier.java` attempts to read `meta-data` value from `Manifest` before assigning notification icon to launcher icon.

**Comment by guperrot** [comment]
Hi, thanks for your feature request.

If we implement the feature, we will likely use another key in manifest or another resource key instead since we are not using Firebase SDK anymore.

We don't have any ETA for this feature request.

**Comment by MatkovIvan** [comment]
This is available in version [1.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/1.5.0)
